src/de/jost_net/JVerein/gui/action/SollbuchungOeffnenAction.java

```diff
@@ -0,0 +1,51 @@
+/**********************************************************************
+ * Copyright (c) by Heiner Jostkleigrewe
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,  but WITHOUT ANY WARRANTY; without
+ *  even the implied warranty of  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See
+ *  the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program.  If not,
+ * see <http://www.gnu.org/licenses/>.
+ *
+ * heiner@jverein.de
+ * www.jverein.de
+ **********************************************************************/
+
+package de.jost_net.JVerein.gui.action;
+
+import de.jost_net.JVerein.gui.view.MitgliedDetailView;
+import de.jost_net.JVerein.rmi.Buchung;
+import de.jost_net.JVerein.rmi.Mitglied;
+import de.willuhn.jameica.gui.Action;
+import de.willuhn.logging.Logger;
+import de.willuhn.util.ApplicationException;
+import de.willuhn.jameica.gui.GUI;
+
+import java.rmi.RemoteException;
+
+public class SollbuchungOeffnenAction implements Action {
+    @Override
+    public void handleAction(Object context) throws ApplicationException {
+        if (!(context instanceof Buchung) && !(context instanceof Buchung[]))
+        {
```

Comment:
Hier bitte auch auf ungleich null anfragen sonst gibt es bei null Exceptions.
Wie ich in #418 geschrieben habe wird bei mir öfters mal eine Action mit null aufgerufen.

Reply:
`null instance of Class` wertet sich sich immer zu `null` aus.

Reply:
Vgl.: https://docs.oracle.com/javase/specs/jls/se11/html/jls-15.html#jls-15.20.2

Reply:
Laut Internet hast du Recht.